src/modules/cbm/CbmInstrument.cpp

```diff
@@ -105,14 +110,38 @@ void CbmInstrument::processMonitorReadouts(void) {
 
     CbmType type = static_cast<CbmType>(readout.Type);
 
-    uint32_t PixelId = 1;
     if (type == CbmType::IBM) {
-      PixelId = readout.NPos & 0xFFFFFF; // Extract lower 24 bits
```

Comment:
You could consider to use a #define  like
#define PIXEL_MASK 0x00FFFFFF 
or similar